### ISSUE TITLE
outputs mapping

### ISSUE THREAD

**Issue by OverGreenWaters** [lisaterumi/nlp-portuguese-postagger#1]
27 classes
table of tags lists 22
what is the mapping from argmax(outputs.logits, dim=2) to the pos?

**Comment by lisaterumi** [comment]
Hello,

Please find the list of class mappings at the following link: 

https://huggingface.co/lisaterumi/postagger-portuguese/blob/main/config.json

If you have any questions, feel free to reach out.

Elisa
